appveyor.yml

```diff
@@ -32,7 +32,7 @@ before_test:
   - ctest -V
 
 artifacts:
-  - path: OpenMEEG-*-x64--static.exe
+  - path: c:\projects\openmeeg\build\OpenMEEG-*-x64--static.exe
```

Comment:
l24-27 seems strange to me. 
Normally, cpack has to be called in the directory `c:\projects\openmeeg\build\`, it seems the case but `move` move the files in the same place. If you change the . by c:/ I think it will work.